app/app/src/main/kotlin/com/hedvig/app/feature/referrals/ui/tab/ReferralsGraph.kt

```diff
@@ -217,6 +218,72 @@ private fun ForeverScreen(
     refreshingOffset = PullRefreshDefaults.RefreshingOffset + systemBarInsetTopDp,
   )
   val incentive = uiState.incentive
+
+  val sheetState = rememberModalBottomSheetState()
+  val coroutineScope = rememberCoroutineScope()
+
+  LaunchedEffect(uiState.showEditCode) {
+    coroutineScope.launch {
+      if (uiState.showEditCode) {
+        sheetState.show()
+      } else {
+        sheetState.hide()
+      }
+    }
+  }
+
+  if (uiState.showEditCode) {
+    ModalBottomSheet(
+      onDismissRequest = {
+        hideEditCode()
+      },
+      // todo use "https://github.com/c5inco/smoother" for a top only squircle shape here
+      sheetState = sheetState,
+      tonalElevation = 0.dp,
+    ) {
+      Text(
+        text = stringResource(id = hedvig.resources.R.string.referrals_change_change_code),
+        textAlign = TextAlign.Center,
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(horizontal = 16.dp),
+      )
+      Spacer(Modifier.height(32.dp))
+      HedvigTextField(
```

Comment:
This text field gets a white container color. Like this: 
<img width="341" alt="Screenshot 2023-07-27 at 14 40 09" src="https://github.com/HedvigInsurance/android/assets/5929732/f5d44bf5-b5eb-447c-99d9-ba2b57da57ef">

Any idea on how to fix that??

Reply:
Hmm do you meant that the text field has the exact same color as the sheet background?

Reply:
Exactly

Reply:
I think the issue is actually that the bottom sheet uses the same color as the text field.
While the text field expects the colors behind it to be `background` but the material sheet is probably using `surface`, or maybe `surfaceVariant`, which the text field may also be using.

Maybe see if you can use `surface` for the MaterialBottomSheet itself and see how that looks and go from there?

That's my assumption at least, if I'm wrong we can see what else we can do.

Reply:
Right, I meant to write "Maybe see if you can use **background**" 🤦‍♂️
Glad it worked!